### PR TITLE
Added Icons in Navbar

### PR DESCRIPTION
**Pull request by mohak852**

### CODE REVIEW

**Comment by sara-02** [comment]
Ideal format is `uiser_name:forked_branch_name` as shown here:
Screenshot from 2020-05-25 23-53-50

**Comment by sara-02** [comment]
@mohak852 This is correct. Can you please share a screen here as well.
Sorry for the inconvenience caused.

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
-
+import slack from './slack.svg';
+import youtube from './youtube.svg';
+import twitter  from './twitter.svg';
+import github from './github.svg';
 class Navbar extends Component {
     render() {
         return (
             <div className="topnav">
+                <img src={slack} className="slack"/>
```
Comment (Roshanjossey) [inline src/components/Navbar/Navbar.jsx:11]: Please group the icons and text together and icon should also be clickable.

I'd suggest

```
<a>
  <img/>
  <span>text</span>
</a>
```

File: src/components/Navbar/Navbar.css [hunk 2]
```diff
@@ -21,9 +21,44 @@
...
 .topnav a.active {
     background-color: #4CAF50;
     color: white;
 }
+.slack{
+    display: inline-block;
+    vertical-align:middle;
+    flex: 1;
+    margin-top: 10px;
+    width: 90px;
```
Comment (Roshanjossey) [inline src/components/Navbar/Navbar.css:40]: Please make the size of icons close to the size of text.

File: src/components/Navbar/Navbar.css [hunk 3]
```diff
@@ -21,9 +21,44 @@
...
+    height: 50px;
+    resizeMode: 'contain';
+}
+.youtube{
+  vertical-align:middle;
+    margin-top: 10px;
+    width: 70px;
+    height: 50px;
+    resizeMode: 'contain';
```
Comment (Roshanjossey) [inline src/components/Navbar/Navbar.css:49]: `resizeMode` is not a CSS style. Not sure why it's here.

File: src/components/Navbar/Navbar.css [hunk 4]
```diff
@@ -21,9 +21,44 @@
...
+    margin-top: 10px;
+    vertical-align:middle;
+    width: 50px;
+    height: 50px;
+    resizeMode: 'contain';
+}
+.twitter{
+    margin-top: 10px;
+    width: 50px;
+    vertical-align:middle;
```
Comment (Roshanjossey) [inline src/components/Navbar/Navbar.css:61]: multiple styles are repeated between these classes,

`margin-top`, `vertical-align` etc. Please put this in a single class .

**Comment by Roshanjossey** [comment]
With current changes
image

Some changes I tried out.
image

**Comment by mohak852** [comment]
Screenshot from 2020-05-28 04-28-52
BRO IS THAT OK!

**Comment by mohak852** [comment]
I have updated all of the changes. You can review that

File: src/components/Navbar/Navbar.css [hunk 5]
```diff
@@ -13,17 +13,50 @@
...
+    width: 60px;
+    height: 50px;
+    display: inline-block;
+}
+.twitter{
+    margin-top: 10px;
+    width: 60px;
+    vertical-align:middle;
+    height: 50px;
+    fill: white;
```
Comment (Roshanjossey) [inline src/components/Navbar/Navbar.css:60]: We don't need `fill: white` here. It's been added to the SVG

**Comment by Roshanjossey** [comment]
@mohak852 , I've resolved all the suggestions that has been addressed. Could you address the remaining ones as well (size of icons, putting common styes in a different class). Feel free to ask if you have some questions.

**Comment by mohak852** [comment]
Bro should I make icons smaller than this and if so please can you send me the accurate pixels I will adjust accordingly. 


Thanks

**Comment by mohak852** [comment]
Bro I Have made all the changes and made the CSS code more shorten using the same classname now you can check the changes in that and if you want any other changes you can tell me.

**Comment by mohak852** [comment]
Screenshot from 2020-05-29 05-56-59

**Comment by Roshanjossey** [comment]
Thank you for the changes @mohak852 . Meging

**Pull request merged**